tools/buildsys/src/gomod/error.rs

```diff
@@ -0,0 +1,33 @@
+use std::path::PathBuf;
+
+use snafu::Snafu;
+
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub(super)))]
+pub(crate) enum Error {
+    #[snafu(display("Failed to start command: {}", source))]
+    CommandStart { source: std::io::Error },
+
+    #[snafu(display("Failed to execute command: 'docker {:?}'", args))]
+    DockerExecution { args: String },
```

Comment:
`String` implements `Display`, so you don't have to use the `Debug` impl:
```suggestion
    #[snafu(display("Failed to execute command: 'docker {}'", args))]
    DockerExecution { args: String },
```